ROADMAP.md

```diff
@@ -0,0 +1,35 @@
+# Roadmap
+
+To better communicate with the community, we plan to share our main focuses each quarter in this
+document.
+
+# Q4 2019 (October, November, December)
+
+We have a few major themes for this quarter:
+
+1.  Breaking out models into their own repositories.  By doing so we hope to increase community
+    engagement and ownership of these model repositories, allowing us to focus more on the core
+    library.  We already have a good start on this with the following repositories:
+
+    a.  [Semantic Parsing](https://github.com/allenai/allennlp-semparse)
+    
+    b.  [Language Modeling](https://github.com/allenai/allennlp-language-modeling)
```

Comment:
I would list reading comprehension here instead, it's a lot closer to ready.  The LM one is just a skeleton so far.  I would add this after these lines:

```
We will test out this idea of splitting up the repositories with these two first,
fixing any issues with them by the end of this quarter.  Around the end of
this quarter or early next quarter, we will finish the process with the rest of
the models, making repositories around language modeling, coreference
resolution, sequence tagging, parsing, etc.
```

Reply:
And you can add a reference to the ticket that @brendan-ai2 mentioned.